docs/docs/manual/wikidata.md

```diff
@@ -6,28 +6,240 @@ sidebar_label: Wikidata
 
 ## Overview
 
+OpenRefine includes powerful ways to both pull data from Wikidata and add data to it. 
 
+OpenRefine’s connections to Wikidata were formerly an optional extension, but are now installed automatically with the downloadable package. The Wikidata extension can be removed manually by navigating to your OpenRefine installation folder, and then looking inside `webapp/extensions/` and deleting the `wikidata` folder inside. 
 
-*   Former extension, now core [but you can remove it].
-*   Account required
+You do not need a Wikidata account to reconcile your local OpenRefine project to Wikidata. If you wish to upload your cleaned dataset to Wikidata, you will need to register an account, and authorize OpenRefine with that account. 
 
+The best source for information about how OpenRefine works with Wikidata is [on Wikidata itself, under Tools](https://www.wikidata.org/wiki/Wikidata:Tools/OpenRefine). This has tutorials, guidelines on editing, and spaces for discussion and help.
 
-## Features
+Everything true about the way OpenRefine works with Wikidata is also true about any other Wikibase. Anything built on the same Wikibase platform will offer a reconciliation endpoint that works the same. To find a Wikibase reconciliation URL, 
```

Comment:
Whoops! We left off on our discussion of this because the wikibase connections are in flux right now (https://phabricator.wikimedia.org/T197588). 
I think we may leave this bit off for now.